installer_test.go

```diff
@@ -2,15 +2,17 @@ package install
 
 import (
 	"context"
+	"os"
```

Comment:
No longer needed with `t.Setenv()` 👍 

```suggestion
	"os"
```